applications/ContactStructuralMechanicsApplication/python_scripts/custom_sympy_fe_utilities.py

```diff
@@ -1,28 +1,66 @@
-from sympy import *
-
-def DefineMatrix( name, m,n ):
-    return Matrix( m,n, lambda i,j: var(name+'_%d_%d' % (i,j)) )
-
-def DefineSymmetricMatrix( name, m,n ):
+import sympy
+
+def GetSympyVersion():
+    """ This method returns the current Sympy version
+    """
+    return sympy.__version__
+
+def DefineMatrix( name, m, n ):
+    """ This method defines a symbolic matrix
+
+    Keyword arguments:
+    name -- Name of variables.
+    m -- Number of rows.
+    n -- Number of columns.
+    """
+    if float(GetSympyVersion()) <= 1.2:
```

Comment:
@loumalouomega this works just fine as long as you use it just as a symbol no?

it fails when you need more complex behaviour but not for most of simple usages

Reply:
Yes, I am working in the actual solution. Can you approve?